vars/daosPackagesVersion.groovy

```diff
@@ -73,23 +85,6 @@ String call(String distro, String next_version) {
         return rpm_version_cache + rpm_dist(_distro)
     }
 
-    /* what's the query to get the highest 1.0.x package?
-    if (target_branch == "weekly-testing-1.x") {
-        return "release/0.9"
-    }
```

Comment:
You cannot remove this.  This is common code that needs to be able to run against historic releases.